enumerables.rb

```diff
@@ -0,0 +1,269 @@
+module Enumerable
+  def my_each
```

Comment:
HINT 💡 
If you address range class here you would automatically address in other methods. Please do this